src/test/java/htmlpublisher/workflow/PublishHTMLStepTest.java

```diff
@@ -114,7 +114,7 @@ public void publishReportOnBuildLevel() throws Exception {
         
         // Run the project
         final HtmlPublisherTarget target = new HtmlPublisherTarget
-            ("testReport", TEST_REPORT_DIR, "index.html", true, false, false);
+            ("testReport", TEST_REPORT_DIR, "index.html","index", true, false, false);
```

Comment:
Looking better, thanks! Still a space needed here before "index" and three other occurrences in this file :)